EKF/common.h

```diff
@@ -80,10 +80,12 @@ struct flow_message {
 };
 
 struct ext_vision_message {
-	Vector3f posNED;	///< measured NED position relative to the local origin (m)
-	Quatf quat;		///< measured quaternion orientation defining rotation from NED to body frame
+	Vector3f posNED;	///< XYZ position in earth frame (m) - Z must be aligned with down axis
```

Comment:
I think it is now more appropriate for these to be renamed to pos and vel, without the NED prefixes.

Reply:
Yes, that is true